frontend/src/components/ParticipantInfo.js

```diff
@@ -316,7 +366,12 @@ const ParticipantInfo = observer(() => {
                         name="hasInsurance"
                         className={classes.group}
                         value={participant.hasInsurance}
-                        onChange={e => changeParticipant(e, "hasInsurance")}
+                        onChange={e =>
+                          setParticipant({
+                            ...participant,
+                            hasInsurance: e.target.value,
+                          })
+                        }
                         style={{ display: "inline" }}
                       >
                         <FormControlLabel
```

Comment:
Can we change the values for these radio buttons to `value={true}` and `value={false}` since that's how the Participant object is being populated?